src/main/psxsdk/libsnd/ssclose.c

```diff
@@ -1,7 +1,33 @@
 #include "common.h"
 #include "libsnd_i.h"
 
-INCLUDE_ASM("main/nonmatchings/psxsdk/libsnd/ssclose", _SsClose);
```

Comment:
You can just use a `for` loop: https://decomp.me/scratch/LEADb

also `var_a2` can be renamed `i`